src/bandersnatch_filter_plugins/latest_name.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from operator import itemgetter
+
+from packaging.version import parse
+
+from bandersnatch.filter import FilterReleasePlugin
+
+logger = logging.getLogger("bandersnatch")
+
+
+class LatestReleaseFilter(FilterReleasePlugin):
+    """
+    Plugin to download only latest releases
+    """
+
+    name = "latest_release"
+
+    def initialize_plugin(self):
+        """
+        Initialize the plugin reading patterns from the config.
+        """
+        self.keep = 0  # default, keep 'em all
+        try:
+            self.keep = int(self.configuration["latest_release"]["keep"])
+        except KeyError:
+            pass
+        except ValueError:
+            pass
+        if self.keep > 0:
+            logger.info(f"Initialized latest releases plugin with keep={self.keep}")
+
+    def filter(self, releases):
+        """
+        Filter the dictionary {(release, files)}
+        """
+        if self.keep == 0:
+            return releases
+
+        versions = map(lambda v: (parse(v), v), releases.keys())
+        latest = sorted(versions)[-self.keep :]  # noqa
```

Comment:
What's the noqa for? Can we 
```
# noqa: ERROR_NUM
e.g.
# noqa: T484
```

Reply:
Hi, thanks for all the responses :)

I had to add #noqa because [flake8](http://flake8.pycqa.org/) (and me) does not like the space between `[-self.keep` and `:]`, but [Black](https://black.readthedocs.io/) adds it... 

I'll read the other comments tomorrow.

Reply:
changed for # noqa: E203

I think it's a bug in Black because `[-keep:]` is not reformated into `[-keep :]`. Or I miss something :-(